pkg/koordlet/util/system/cgroup_driver_linux.go

```diff
@@ -122,3 +124,7 @@ func GuessCgroupDriverFromKubeletPort(port int) (CgroupDriverType, error) {
 	klog.Infof("Cgroup driver is not specify in kubelet config file, use default: '%s'", kubeletDefaultCgroupDriver)
 	return kubeletDefaultCgroupDriver, nil
 }
+
+func IsUsingCgroupsV2() bool {
+	return libcontainercgroups.IsCgroup2UnifiedMode()
```

Comment:
The mountpoint of the host cgroups in the koordlet is configurable (e.g. `/host-cgroup`), while the `IsCgroup2UnifiedMode()` checks `/sys/fs/cgroup` by default. Perhaps we need a parameterized `IsCgroup2Mode()`.